Src/libCZI/Doc/building_libCZI.markdown

```diff
@@ -36,6 +33,32 @@ The same steps as above will build the code - go into the folder where the libCZ
     cmake --build .
 
 
+Configurations
+--------------
+
+The CMake-file defines the following options for customizing the build:
+
+option                                     | description
+-------------------------------------------|------------
+LIBCZI_BUILD_UNITTESTS                     | Whether to build the unit-tests for libCZI. Default is **ON**.
+LIBCZI_BUILD_CZICMD                        | Whether to build the test- and sample-application CZICmd. Default is **OFF**.
+LIBCZI_BUILD_DYNLIB                        | Whether to build the dynamic link libaray for libczi. Default is **ON**.
+LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_EIGEN3 | Whether to use an existing Eigen3-library on the system (included via find_package). If this is OFF, then a copy of Eigen3 is downloaded as part of the build. Default is **OFF**.
+LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_ZSTD   | Whether to use an existing zstd-library on the system (included via find_package). If this is OFF, then a copy of zstd is downloaded as part of the build. Default is **OFF**.
+
+If building CZICmd is desired, then running CMake with this command line will enable building CZICmd:
+    
+    cmake .. -DLIBCZI_BUILD_CZICMD=ON
+
+Building CZICmd requires the external package RapidJSON to be available. In addition, on Linux the packages ZLIB, PNG and (optionally) Freetype are needed.
```

Comment:
Can you maybe provide a few more information on how to make this work in Visual Studio 2022? I somehow only managed to get it work through the shell following https://github.com/microsoft/vcpkg#quick-start-windows and then executing the steps in [cmake.yml](https://github.com/ZEISS/libczi/blob/main/.github/workflows/cmake.yml).

Reply:
...will do, I promise - in another PR